src/neptune/internal/utils/images.py

```diff
@@ -228,6 +220,39 @@ def _get_numpy_as_image(array):
     )
 
 
+def _scale_array(array: np.ndarray) -> np.ndarray:
+    min_value = np.min(array)
+    max_value = np.max(array)
+
+    if min_value >= 0 and max_value <= 1:
+        return array
+
+    # Avoid division by zero if all values are the same
+    if min_value == max_value:
+        scaled_array = np.zeros_like(array)
+    else:
+        scaled_array = (array - min_value) / (max_value - min_value)
```

Comment:
Are you sure about this approach? This will change the brightness of the image if I understand. This may work for ML-purposes to normalize images as an input to some model but I would rather just divide by `255` to make the image consistent.

Reply:
or just divide by `maxabs` value. Dividing by 255 makes sense if array is of type `uint8`